### PR TITLE
Can now ingest from FM and get a grid of IDs

### PR DESCRIPTION
**Pull request by mccalluc**
@afred : please review

### CODE REVIEW

File: app/views/shared/_search.html.erb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<form action="/catalog">
+  <img src="/icon-search.png"/>
+  <input placeholder="Search over 10,000 WGBH clips online" 
+         onchange="this.form.submit();" name="q"/>
+</form>
```
Comment (afred) [inline app/views/shared/_search.html.erb]: @mccalluc The `onchange` event will fire if you type a search and then clicks or tabs away from the input. I don't think we want that. Try this maybe:
```html
<form action="/catalog">
  <input type="image" src="/icon-search.png"/ alt="Search">
  <input placeholder="Search over 10,000 WGBH clips online" name="q"/>
</form>
```

File: app/views/catalog/_document_list.html.erb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<% # container for all documents in index list view -%>
+<div id="documents" class="documents-<%= document_index_view_type %>">
+  <% documents.in_groups_of(4, false) do |document_group| %>
+    <div class="row">
+      <%= render document_group, :as => :document %>
+    </div>
+  <% end %>
+</div>
```
Comment (afred) [inline app/views/catalog/_document_list.html.erb]: @mccalluc I refactored this. See latest commit. May want to confirm visually before pushing. I don't have an index, and thus no search results.

File: app/views/catalog/_document_list.html.erb [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+  <% documents.in_groups_of(4, false) do |document_group| %>
+    <div class="row">
+      <% document_group.each do |document| %>
+        <div class="col-md-3">
+          <%= render document %>
+        </div>
+      <% end %>
+    </div>
+  <% end %>
+</div>
```
Comment (afred) [inline app/views/catalog/_document_list.html.erb]: @mccalluc I refactored this. See latest commit. May want to confirm visually before pushing. I don't have an index, and thus no search results.

**Comment by mccalluc** [comment]
@afred: Your changes look fine. Thanks. Re-review this handling of the search button.

**Comment by mccalluc** [comment]
Ack: Actually I hadn't fully updated: your changes are breaking... hold on.

File: app/views/catalog/_document_list.html.erb (hunk 3 above)
Comment (afred) [inline app/views/catalog/_document_list.html.erb]: @mccalluc so the problem was just with the `document_counter`?

Comment (mccalluc) [inline app/views/catalog/_document_list.html.erb]: @afred: Sort-of... except that when I tried just passing it as a local variable, that didn't work. For thinking about how to go forward, I think we should clarify whether for us it just makes the most sense to gut the original templates and keep our code as simple as possible... or if we want to preserve their structure as much as possible. I don't know.

**Pull request merged**